.github/workflows/main.yml

```diff
@@ -225,6 +225,21 @@ jobs:
       make-arguments: ${{ github.event.inputs.make-arguments }}
     if: needs.prepare.outputs.linux-x64-variants == 'true'
 
+  build-linux-x64-static:
+    name: linux-x64-static
+    needs: prepare
+    uses: ./.github/workflows/build-linux.yml
+    with:
+      platform: linux-x64
+      make-target: 'static-jdk-image'
+      # There are issues with fastdebug static build in GHA due to space limit.
+      # Only do release build for now.
+      debug-levels: '[ "release" ]'
+      gcc-major-version: '10'
+      configure-arguments: ${{ github.event.inputs.configure-arguments }}
+      make-arguments: ${{ github.event.inputs.make-arguments }}
+    if: needs.prepare.outputs.linux-x64 == 'true'
```

Comment:
Ah, one more trouble: this job produces bundles. But that bundle name is just `linux-x64`, which overrides the standard bundle. See how `build-linux-x64-static-libs` overrides `bundle-suffix:`, do the same here. E.g. `bundle-suffix: -static`?

Reply:
> Ah, one more trouble: this job produces bundles. But that bundle name is just `linux-x64`, which overrides the standard bundle. See how `build-linux-x64-static-libs` overrides `bundle-suffix:`, do the same here. E.g. `bundle-suffix: "-static"`?

The current `build-linux-x64-static` job doesn't produce any bundles. 

I do agree with your point on overriding the bundle name, as we probably want to include bundle creation in the job at some later point. I actually ran into GHA failure initially when I used the `build-linux.yml` for the job due to bundle name conflict before your https://github.com/openjdk/jdk/pull/23715 change. Updating to include your suggestion for the bundle name...